extractor/nft_test.go

```diff
@@ -276,7 +276,7 @@ func TestSendNftWithPrice(t *testing.T) {
 	require.Len(t, incomeDetailsRes.IncomeDetails, 2)
 	require.Equal(t, nfts[0].ClassId, incomeDetailsRes.IncomeDetails[0].ClassId)
 	require.Equal(t, nfts[0].NftId, incomeDetailsRes.IncomeDetails[0].NftId)
-	require.Equal(t, ADDR_02_LIKE, incomeDetailsRes.IncomeDetails[0].Owner)
+	require.Equal(t, ADDR_01_LIKE, incomeDetailsRes.IncomeDetails[0].Owner)
```

Comment:
This field is confusing since there is NftId right next to it, maybe we don't need Owner in response at all

Reply:
Yeah, probably don't have to include `owner` in response.